Lib/test/test_wsgiref.py

```diff
@@ -14,6 +14,7 @@
 from socketserver import BaseServer
 from platform import python_implementation
 
+import io
```

Comment:
Nit: `StringIO` is already imported in line 3 :) But I don't want to bother you with my nitpicks. I can change it before merging.

Reply:
Fixed it. :-) Thanks again for reviewing and approving. Looking forward to stop seeing these stack traces in 3.8.

Reply:
Thanks! I'm planning to backport both PRs to 3.7.

Reply:
Thank you both, on behalf of all of us out here. :) Look very much forward to the 3.7 backport for now, and 3.8 down the road.